circuits/batch_update_state_tree.circom

```diff
@@ -0,0 +1,124 @@
+include "./update_state_tree.circom";
+
+template BatchUpdateStateTree(
+  depth,
+  vote_options_tree_depth,
+  batch_size
+) {
+  // params:
+  //    depth: the depth of the state tree and the command tree
+  //    vote_options_tree_depth: depth of the vote tree
+  //    batch_size: that we're processing
+
+  // Output: New state tree root
+  signal output root;
+
+  // Input(s)
+  signal input coordinator_public_key[2];
+
+  var message_length = 11;
+  signal input message[batch_size][message_length];
+
+  // Select vote option index's weight
+  // (a.k.a the raw value of the leaf pre-hash)
+  signal private input vote_options_leaf_raw[batch_size];
```

Comment:
This is also supplied by the coordinator, right?

Reply:
Yeap! Anything to do with `vote_options` is supplied by the coordinator